src/rhsmlib/dbus/objects/register.py

```diff
@@ -298,21 +307,33 @@ def _remove_enable_content_option(options: dict, default: bool = False) -> bool:
         enable_content = options.pop("enable_content")
         return is_true_value(enable_content)
 
-    @staticmethod
-    def _enable_content(uep: "UEPConnection", consumer: dict) -> None:
+    def _enable_content(self, uep: "UEPConnection", consumer: dict) -> None:
         """Try to enable content: refresh SCA entitlement certs in SCA mode."""
         content_access: str = consumer["owner"]["contentAccessMode"]
 
         if content_access == "entitlement":
             log.error("Entitlement content access mode is not supported")
         elif content_access == "org_environment":
+            log.info("Enabling generating content (redhat.repo) from SCA certificate in rhsm.conf.")
+            self.config["rhsm"]["manage_repos"] = "1"
+            self.config.persist()
             log.debug("Refreshing since 'enable_content' is true.")
             service = EntitlementService(uep)
             # TODO: try get anything useful from refresh result. It is not possible atm.
             service.refresh(remove_cache=False, force=False)
         else:
             log.error(f"Unable to enable content due to unsupported content access mode: '{content_access}'")
 
+    def _disable_content(self) -> None:
+        """
+        Try to disable generating content from SCA certificates. This method
```

Comment:
I suggest updating this comment

From:
`sets value manage_repos` 


```
        Try to disable generating content from SCA certificates. This method
        sets value manage_repos to 0 in rhsm.conf to avoid generating redhat.repo
        file from SCA certificates by subscription-manager or rhsmcertd.
```

To:

`sets the value of manage_repos`

As follows:

```
        Try to disable generating content from SCA certificates. This method
        sets the value of manage_repos to 0 in rhsm.conf to avoid generating redhat.repo
        file from SCA certificates by subscription-manager or rhsmcertd.
```